src/rules/converters/tests/no-shadowed-variable.test.ts

```diff
@@ -1,4 +1,8 @@
-import { convertNoShadowedVariable } from "../no-shadowed-variable";
+import {
+    convertNoShadowedVariable,
+    SELECTIVE_DISABLE_NOTICE,
+    UNDERSCORE_DISABLE_NOTICE,
+} from "../no-shadowed-variable";
```

Comment:
Similarly, I don't see why these kinds of changes would be necessary to drop NodeJS 8 support. Let's revert.